spec/cluster_holding_spec.rb

```diff
@@ -117,6 +131,29 @@ def new_submission(holding, date: Date.today)
       h2 = build(:holding, ocn: h.ocn, uuid: h.uuid)
       expect { described_class.new(h2).update }.to raise_exception(/same UUID/)
     end
+
+    it 'skips duplicative holdings for a member in "large clusters"' do
+      c.ocns = [Services.large_clusters.ocns.first]
+      c.save
+      dupes = [build(:holding, organization: "umich", ocn: c.ocns.first),
+               build(:holding, organization: "umich", ocn: c.ocns.first)]
+      described_class.new(dupes).update
+      cluster = Cluster.first
+      expect(cluster.holdings.count).to eq(1)
+    end
+
+    it 'updates the date_received for "large clusters"' do
```

Comment:
Does this test something different from the "updates multiple existing holdings if cluster.large?" spec I commented on above?

Reply:
Previous test was both existing dupes get updated by a single holding. This test is a single holding being updated by a batch of dupes. The exact motivation escapes me, but I assume I managed to screw this up at some point along the way.

Reply:
OK. I was assuming that large cluster should always imply no dupes. What are the consequences of potentially ending up with dupes in a large cluster? It shouldn't matter for ETAS overlap reports, since we force copy_count to 1 for that; does it make a difference for cost calculation? I suppose we could have a "dedupe" function to clean up large clusters post-merge/update. Again, let's discuss Friday.

Reply:
It might be worth a spec to verify that multiple reported holdings for a serial do not affect cost calculation.

Reply:
We already have a spec for the cost report that multiple holdings do not affect h-share: https://github.com/hathitrust/holdings-backend/blob/master/spec/cost_report_spec.rb#L222

So I think we can go ahead and merge this as-is.